python/domains.py

```diff
@@ -8,7 +8,7 @@
 from sdk.python.util.google import reconstitute_conversion_script
 from sdk.python.util.google import extract_script_parameters
 from sdk.python.util.brand_util import PLATFORM_MASCOT_ICON
-from sdk.python.util.templates import compile_template
+from common.themes.templates import compile_template
```

Comment:
oh, this doesn't work, there's no common/ directory in this repo

Reply:
@DanielCollins As discussed last night we'll move compile_template out of sdk and into the backend?

Reply:
yeah

Reply:
i can work on that on monday